test/index.js

```diff
@@ -907,4 +908,11 @@ describe('transformPhrase', function () {
     expect(Polyglot.transformPhrase(english, { smart_count: 2 }, 'fr')).to.equal('2 Names');
     expect(Polyglot.transformPhrase(english, { smart_count: 2 }, 'fr')).to.equal('2 Names');
   });
+
+  it('selects the correct plural form based on smart_count', function () {
+    expect(Polyglot.transformPhrase(chinese, { smart_count: 0 }, 'zh')).to.equal('0 个');
+    expect(Polyglot.transformPhrase(chinese, { smart_count: 1 }, 'zh')).to.equal('1 个');
+    expect(Polyglot.transformPhrase(chinese, { smart_count: 2 }, 'zh')).to.equal('2 个(plural)');
+    expect(Polyglot.transformPhrase(chinese, { smart_count: 3 }, 'zh')).to.equal('3 个(plural)');
```

Comment:
thanks! it'd be great if you could use actual chinese logograms here instead of the "(plural)", so that the phrasing makes sense in the native language instead of just demonstrating the technical functionality :-)

Reply:
I just use `plural` to easily notice the pluralization. There is the same without logograms between singular and plural in fact but for program it's logical to know if pluralization. Is it ok to use the translation of `plural` in Chinese instead of `plural`?

Reply:
There is no difference in form but a difference in logic. I have mentioned above that distinguishes if pluralization in logic throught the key of return.

Reply:
I'm still confused, thanks for bearing with me. The only thing polyglot is concerned with is outputting translated prose with the proper pluralization - if the outputted text is identical for a count of 1 vs 2, say, then any other logic could be done at the callsite, since you have the count available there. What am I missing?

Reply:
Of course, it can be done this way with only one branch, but usually i18n translations are standardized translations based on English documentation, where the branches for singular and plural are different. 

Since Chinese also has a logical concept of pluralization, it is normal to distinguish and handle the output of singular and plural. If I had not participated in the Chinese translation of Navidrome, I would not have realized that `polyglot.js` does not distinguish between singular and plural output. 

Therefore, even if the singular and plural forms are the same in appearance, there is still a logical need to distinguish between singular and plural. These are just some of my trivial thoughts, and I don't know what other developers think. I apologize for any inconvenience this may have caused you.

Reply:
I'm not convinced by "usually" - at airbnb, we didn't have them based on English, since there's many languages with more than 2 branches.

Can you help me understand the logical need? If the prose is the same, why does one need to distinguish?

Reply:
You focus on the result, and I focus on the process. 

My understanding is to distinguish between singular and plural to choose the branch template like `this is single.` or `this is multiple, the amount is %{smart_count}.`, instead of having no choice but to use only one template like `the amount is %{smart_count}`, which seems like there is no plural concept and lack of flexibility in Chinese compared to other languages with branches. 

I compromise, perhaps life doesn't need to be so complicated.